meta-networking/recipes-connectivity/snort/snort3_git.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "snort3 - a free lightweight network intrusion detection system for UNIX and Windows."
+HOMEPAGE = "http://www.snort.org/"
+SECTION = "net"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://COPYING;md5=78fa8ef966b48fbf9095e13cc92377c5"
+
+DEPENDS = "luajit libdaq libdnet flex hwloc libpcap libpcre zlib libtirpc"
+RDEPENDS:${PN} += "bash"
```

Comment:
Can we remove this direct dependency ? perhaps convert the shell to not need bash extentions or move the concerning scripts to package of its own.

Reply:
I tried to remove this dependency by creating a patch for the bash script but there are a lot of changes to be made in this script and I don't need it for my use case.

Reply:
OK, can be move the script to a package of its own. ? something like below

PACKAGES += "${PN}-scripts"

FILES:${PN}-scripts = "/path/to/script/name"

RDEPENDS:${PN}-scripts = "bash"

Reply:
sure, I'll try making this change, Also can I push all my changes in one single commit rather than making a commit for each recipe?

Reply:
Usually one commit per recipe is good